main.js

```diff
@@ -76,7 +76,7 @@ export class Main extends Scene {
         // opposite side
         this.wall.render(context, program_state, length, 5, 3, -1.5, Mat4.translation(0, -.5, 1.5 - depth));
         this.car_end.render(context, program_state, depth * 2, 12, 10.75, Mat4.translation(-length, -.5, 0).times(Mat4.rotation(Math.PI / 2, 0, 1, 0)));
-        this.handlebars.render(context, program_state, t, Math.atan(this.acceleration/9.8), this.trainMove,  Mat4.translation(0, 9, 2 - depth));
+        this.handlebars.render(context, program_state, t, Math.atan(this.acceleration/9.8), this.trainMove, 13, 3.5, length * 2, Mat4.translation(0, 9, 2 - depth));
```

Comment:
We can have `HandleBar.js` dynamically determine the spacing required based off the number off handles and length of the bar. So we no longer need this parameter. 

```suggestion
        this.handlebars.render(context, program_state, t, Math.atan(this.acceleration/9.8), this.trainMove, 13, length * 2, Mat4.translation(0, 9, 2 - depth));
```